style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: green;
+}
```

Comment:
- [ ]  Kindly make use of the `class` name created in the HTML header to style your HTML content instead of using an `h1` tag for the styling
for example: 👇
``` CSS
.head {
  color: green;
}
```

Reply:
Thank you for the example, I will change immediately